sample/src/main/java/ch/sonect/sdk/shop/integrationapp/CacheManager.kt

```diff
@@ -0,0 +1,47 @@
+package ch.sonect.sdk.shop.integrationapp
+
+import android.content.Context
+
+class CacheManager(context: Context) {
```

Comment:
Why do we need to save them?

Reply:
the details?

Reply:
@igor-giziy Testers asked it to make it easier. They enter different partner creds and in order to test it back and forth it will be usefull to save latest.